roles/kubernetes/preinstall/tasks/main.yml

```diff
@@ -109,6 +117,10 @@
 - name: Install epel-release on RedHat/CentOS
   shell: rpm -qa | grep epel-release || rpm -ivh {{ epel_rpm_download_url }}
   when: ansible_distribution in ["CentOS","RedHat"] and not is_atomic
+  register: epel_task_result
+  until: epel_task_result|success
```

Comment:
It's |succeeded in newer Ansible versions
http://docs.ansible.com/ansible/playbooks_conditionals.html#the-when-statement